Week2/homework/map_filter.js

```diff
@@ -3,3 +3,21 @@
 const numbers = [1, 2, 3, 4];
 
 // Add your code here
+
+//using filter and map
+const newNumbers = numbers
+    .filter(function(number) {
+    return number % 2 !==0;
+})
+    .map(function(odd) {
+        return odd * 2; 
+    })
+    
+console.log(newNumbers); //[ 2, 6 ]
+
+//filter, map and =>
+const newNumbers = numbers
```

Comment:
Careful here.  Running your code gave this error:
```
/Users/rohann/devel/hack-your-future/homework/javascript2/Fatuma-class18/Week2/homework/map_filter.js:19
const newNumbers = numbers
      ^

SyntaxError: Identifier 'newNumbers' has already been declared
```
You can't redeclare a const.  I like that you were improving on the first version, but you need to remember to use a different name.  I would also suggest that you run your code often, to catch things like this.